episodes/imaging-software.md

```diff
@@ -0,0 +1,486 @@
+```
+Pan - Click and drag
+Zoom - Scroll in/out
+```
+
+## Dimension sliders
+
+Dimension sliders appear at the bottom of the canvas depending on the type of 
+image displayed. For example, if we drag the slider at the bottom of the cells 
+image, we move up and down in this 3D image stack.
+
+![](fig/dim-slider.png){alt="Three screenshots of the cells image in napari, at 
+different z depths"}
+
+Pressing the arrow buttons at either end of the slider steps through one image 
+at a time. Also, pressing the 'play' button at the very left of the slider moves 
+automatically through the stack until pressed again.
+
+We will see in later episodes that more sliders can appear if our image has more 
+dimensions (e.g. time series, or further channels).
+
+## Viewer buttons
+
+The viewer buttons control various aspects of the Napari viewer:
+
+### Console ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/console.svg
+){alt="A screenshot of Napari's console button" height='30px'}
+
+This button opens Napari's built-in python console - we'll look at this in later 
+episodes.
+
+### 2D/3D ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/2D.svg
+){alt="A screenshot of Napari's 2D button" height='25px'}  / ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/3D.svg
+){alt="A screenshot of Napari's 3D button" height='25px'}
+
+This switches the canvas between 2D and 3D display. Try switching to the 3D view 
+for the cells image:
+
+![](fig/cells-3d-napari.png){alt="A screenshot of 3D cells in Napari"}
+
+The controls for moving in 3D are similar to those for 2D:
+```
+Rotate - Click and drag
+Pan - Shift + click and drag
+Zoom - Scroll in/out
+```
+
+### Roll dimensions ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/roll.svg
+){alt="A screenshot of Napari's roll dimensions button" height='25px'} 
+
+This switches the order of dimensions in the viewer. For example, let's switch 
+back to the 2D view for our cells image and press the roll dimensions button 
+multiple times. You'll see that it switches between different orthogonal views 
+(i.e. at 90 degrees to our starting view). Pressing it 3 times will bring us 
+back to the original orientation.
+
+![](fig/roll-dims.png){alt="Three screenshots of the cells image in napari, 
+with different dimension order"}
+
+### Transpose dimensions ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/transpose.svg
+){alt="A screenshot of Napari's transpose dimensions button" height='25px'}
+
+This button swaps the two currently displayed dimensions. Again trying this for 
+our cells image, we see that the image becomes flipped. Pressing the button 
+again brings us back to the original orientation.
+
+![](fig/transpose-dim.png){alt="Two screenshots of the cells image in napari, 
+with dimensions swapped"}
```

Comment:
I wonder if this could be further illustrated by having a red axes and a blue axes in the first image and then transposing them on the second image, just to enforce what transposing does here.

Reply:
I'll use Napari's View > Axes > Axes Visible to do this. It adds a pink / blue arrow to the viewer for each axis.